src/xdocs/config_blocks.xml

```diff
@@ -939,7 +954,13 @@ class Test
   &lt;property name=&quot;tokens&quot; value=&quot;LITERAL_IF, LITERAL_ELSE&quot;/&gt;
 &lt;/module&gt;
         </source>
-
+        <p>Example:</p>
+        <source>
+if (x &gt; 10) { // OK
+    return true;
+} else // violation, single-line statements not allowed without braces
```

Comment:
please align trailing comment vertically to not mix with java code.  

Please apply this to other java code in examples, like at `Next statements won't be violated by check`.

Reply:
Done.